projects/geos/project.yaml

```diff
@@ -0,0 +1,14 @@
+homepage: "https://geos.osgeo.org"
+language: c++
+primary_contact: "mtnclimb@gmail.com"
+auto_ccs :
+- "strk@kbt.io"
+- "lr@pcorp.us"
+- "p.antoine@catenacyber.fr"
+
+fuzzing_engines:
+  - libfuzzer
```

Comment:
Why just libfuzzer, what is going on with afl++, honggfuzz ?

Reply:
Let CI check this

Reply:
Looks good for afl and hongfuzz indeed

Reply:
Now afl++ is failing with `PLEASE submit a bug report to https://bugs.llvm.org/ and include the crash backtrace, preprocessed source, and associated run script` because of tinyxml2...